xml/ay_bigfile.xml

```diff
@@ -8958,6 +8958,68 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 &lt;/users&gt;</screen>
     </example>
 
+    <para>
+     The next example shows a more complete example. The data in
+     <filename>/etc/default/useradd</filename> is used to determine the home
+     directory of the user to be created plus other parameters.
```

Comment:
The following example shows a more complex scenario. System wide default setting from <filename>/etc/default/useradd</filename>, such as the shell or the parent directory for the home directory, are applied.